turbo/app/snapshots_cmd.go

```diff
@@ -1510,6 +1526,39 @@ func doRetireCommand(cliCtx *cli.Context, dirs datadir.Dirs) error {
 	return nil
 }
 
+func doRemoveOverlaps(cliCtx *cli.Context, dirs datadir.Dirs) error {
+	logger, _, _, _, err := debug.Setup(cliCtx, true /* rootLogger */)
+	if err != nil {
+		return err
+	}
+	defer logger.Info("Done")
+	ctx := cliCtx.Context
+
+	db := dbCfg(kv.ChainDB, dirs.Chaindata).MustOpen()
+	defer db.Close()
+	chainConfig := fromdb.ChainConfig(db)
+	cfg := ethconfig.NewSnapCfg(false, true, true, chainConfig.ChainName)
+
+	blockSnaps, borSnaps, caplinSnaps, _, agg, clean, err := openSnaps(ctx, cfg, dirs, 0, db, logger)
+	if err != nil {
+		return err
+	}
+	defer clean()
+	if blockSnaps != nil {
+		blockSnaps.RemoveOverlaps()
```

Comment:
maybe `RemoveOverlaps` method of RoSnapshots must do `hasCoverVisibleFile` check - same as Agg does.